packages/lesswrong/lib/collections/jargonTerms/schema.ts

```diff
@@ -0,0 +1,66 @@
+import { schemaDefaultValue } from '../../utils/schemaUtils';
+
+const schema: SchemaType<"JargonTerms"> = {
+  postId: {
+    type: String,
+    canRead: ['guests'],
+    canCreate: ['members'],
+    nullable: false,
+  },
+  term: {
+    type: String,
+    canRead: ['guests'],
+    canCreate: ['members'],
+    canUpdate: ['members'],
+    nullable: false,
+  },
+  humansAndOrAIEdited: {
+    type: String,
+    allowedValues: ['humans', 'AI', 'humansAndAI'],
+    canRead: ['guests'],
+    optional: false,
+    nullable: false,
+    ...schemaDefaultValue('humans'),
+  },
+  forLaTeX: {
+    type: Boolean,
+    canRead: ['guests'],
+    canCreate: ['members'],
+    canUpdate: ['members'],
```

Comment:
`forLaTeX` probably wants a stricter `canUpdate` - it's not clear that this should be editable by users at all.